en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -1,8 +1,8 @@
-# Configuring a Distributed API-M Deployment
+# Configuring a Distributed API-M Deployment with Gateway and Control Plane
 
 The WSO2 API-M server can be deployed as an [all-in-one deployment]({{base_path}}/install-and-setup/setup/single-node/all-in-one-deployment-overview) or as a distributed deployment. In the distributed setup, the [API-M server profiles]({{base_path}}/install-and-setup/setup/distributed-deployment/product-profiles) are deployed as separate API-M nodes. 
 
-Given below are the API-M nodes you can have in a distributed deployment.
+Given below are the API-M nodes you can have in a distributed deployment with Traffic Manager included within the Control Plane.
```

Comment:
Shall we change this to "Given below are the API-M nodes you can have in a distributed deployment by default." @tharindu1st wdyt?

Reply:
Changed this.